python/lsst/daf/base/propertyContainer/propertyContainerContinued.py

```diff
@@ -443,6 +462,71 @@ def toDict(self):
                 d[name] = v
         return d
 
+    def items(self):
+        for name in self:
+            yield name, _propertyContainerGet(self, name, returnStyle=ReturnStyle.AUTO)
+
+    def values(self):
+        for name in self:
+            yield _propertyContainerGet(self, name, returnStyle=ReturnStyle.AUTO)
```

Comment:
Is it actually okay for `items` and `values` to be generators?  I thought it was expected that they be view containers.

Reply:
It looks like they are okay doing that but the correct thing is to do what collections.abc.Mapping does and return a KeysView and ItemsView object (which we would get for free if we could inherit properly).